.github/workflows/template_gitops.yml

```
@@ -49,11 +49,11 @@ on:
       gitops-user:
         required: false
         type: string
-        default: "Staffbot"
+        default: "staffbase-actions[bot]"
```

```suggestion
        default: "staffbase-actions"
```

I guess the bot suffix is auto attached by github and must not included in the username.